tests/providers/aws/services/rds/rds_instance_security_group_event_subscription/rds_instance_security_group_event_subscription_test.py

```diff
@@ -0,0 +1,212 @@
+from unittest import mock
+
+import botocore
+from boto3 import client
+from moto import mock_aws
+
+from tests.providers.aws.utils import (
+    AWS_ACCOUNT_NUMBER,
+    AWS_REGION_US_EAST_1,
+    set_mocked_aws_provider,
+)
+
+make_api_call = botocore.client.BaseClient._make_api_call
+
+
+class Test_rds_instance__no_event_subscriptions:
+    @mock_aws
+    def test_rds_no_events(self):
+        from prowler.providers.aws.services.rds.rds_service import RDS
+
+        aws_provider = set_mocked_aws_provider([AWS_REGION_US_EAST_1])
+
+        with mock.patch(
+            "prowler.providers.common.provider.Provider.get_global_provider",
+            return_value=aws_provider,
+        ):
+            with mock.patch(
+                "prowler.providers.aws.services.rds.rds_instance_security_group_event_subscription.rds_instance_security_group_event_subscription.rds_client",
+                new=RDS(aws_provider),
+            ):
+                # Test Check
+                from prowler.providers.aws.services.rds.rds_instance_security_group_event_subscription.rds_instance_security_group_event_subscription import (
+                    rds_instance_security_group_event_subscription,
+                )
+
+                check = rds_instance_security_group_event_subscription()
+                result = check.execute()
+
+                assert len(result) == 1
```

Comment:
Please, you need to assert all the fields within result.

Reply:
Adding the check to see if DB Instances exist in the region changes this to not have a result.